inc/log.class.php

```diff
@@ -1274,4 +1287,57 @@ function getRights($interface = 'central') {
       return $values;
    }
 
+   /**
+    * Build export filename for a given item
+    * Format: {translated_itemtype}_{id}_{date}.csv
+    *
+    * @param CommonDBTM $item
+    *
+    * @return string
+    */
+   public static function getExportName(CommonDBTM $item): string {
+      $date = date('Y_m_d', time());
+      return "{$item::getTypeName(1)}_{$item->getId()}_$date.csv";
+   }
+
+   /**
+    * Export specified item logs to CSV.
+    * Logs may be filtered
+    *
+    * @param CommonDBTM $item
+    * @param array      $filter
+    *
+    * @return League\Csv\Writer
+    */
+   public static function exportToCsv(CommonDBTM $item, array $filter): Writer {
+      // Get logs from DB
+      $filter = self::convertFiltersValuesToSqlCriteria($filter);
+      $logs = self::getHistoryData($item, 0, 0, $filter);
+
+      // Remove uneeded rows
+      $logs = array_map(function($log) {
+         unset($log['display_history']);
+         unset($log['datatype']);
+         return $log;
+      }, $logs);
+
+      // Init file
+      $csv = Writer::createFromString('');
```

Comment:
Writer is not very readdable at this point of code, could you do an alias in your "use" with something like 

`use League\Csv\Writer as Csv_Writer;`